eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -102,7 +102,7 @@ object Channel {
 class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId: PublicKey, blockchain: ActorRef, relayer: ActorRef, origin_opt: Option[ActorRef] = None)(implicit ec: ExecutionContext = ExecutionContext.Implicits.global) extends FSM[State, Data] with FSMDiagnosticActorLogging[State, Data] {
 
   import Channel._
-  import nodeParams.keyManager
+  import nodeParams.channelKeyManager
```

Comment:
What about:

```
private val keyManager = nodeParams.channelKeyManager
```

So that all the other changes to this file are unnecessary :)

Reply:
Done